src/deploymentmanager.ts

```diff
@@ -553,6 +558,60 @@ export class DeploymentManager implements IDeploymentManager {
 
         console.log('Copy the following environment variables to /scripts/local/.env file: \n\ %s', `${chalk.cyan(data.join('\n'))}`);
     }
+
+    private setEnvironmentVariables(outputs: any, storageEndpointSuffix: string, environmentFileName: string) {
```

Comment:
printEnvironmentVariables and setEnvironmentVariables has lot of common code. Refactor it into common method

Reply:
yes I was thinking over that